app/styles/partials/_info.scss

```diff
@@ -248,4 +248,58 @@ $packages: (
     width: 30px;
     order: 1;
   }
-}
\ No newline at end of file
+}
+
+.contributors img {
+  border-radius: 50%;
+  border: 2px solid #fff;
+  width: 30px;
+  margin-right: 5px;
+}
+
+/* Tooltip container */
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+/* Tooltip text */
+.tooltip .tooltiptext {
+  visibility: hidden;
+  width: 80px;
```

Comment:
I wouldn't  set strict width here. Hovewer without width set up, there is a problem with a scroll bar appearing inside .info section. `.info` class should have `overflow: hidden;` declaration to overcome this. Also, I wouldn't break tooltip text on multiple columns, but keep it in just one line. To do that there is `white-space: nowrap;` declaration needed. I would also add more padding to .tooltiptext selector (e.g. padding: 7px 20px 8px; - note that bottom padding is a bit longer - this is because text is usually not perfectly centered vertically). We also need to `@include flex-center` mixin here

Reply:
Great tips, thank you! Tooltip looks much better now. However I have one problem - I don't know an easy way to make tooltip visible over right boundary of the parent div. I tried z-index, turning off display grid etc.

It looks this way:
![Screenshot from 2021-11-21 02-35-24](https://user-images.githubusercontent.com/4669074/142745977-eec8c6e9-f1fe-4756-ab67-d15b653c43a9.png)

**Proposal:** 
I used a short spike to find possible options. If you don't have any quick solution I propose to keep breaking tooltip into multiple rows (remove `white-space: nowrap`), so it looks this way:

![Screenshot from 2021-11-21 02-40-00](https://user-images.githubusercontent.com/4669074/142746024-379dce92-8853-4446-9eb1-8672c0d8bf0b.png)


WDYT?

Reply:
My bad, in `.info` class instead of `overflow: hidden;` there should be `overflow: visible`. Let's also remove `overflow-y: auto;` from there (line 29). It should resolve the issue, and white-space: nowrap should work properly now

Reply:
Thanks! Done